engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -47,6 +47,8 @@ def __init__(self, model_id_or_path: str, properties: dict, **kwargs):
         :param properties (dict): other properties of the model, such as decoder strategy
         """
         self.lmi_dist_config = LmiDistRbProperties(**properties)
+        self.model_type = getattr(kwargs.get("model_config", None),
+                                  "model_type", None)
```

Comment:
QQ: I see we are populating model_config, Where do we populate "model_type"

Reply:
model_config here is hugging face config.json which has model_type populated.

Reply:
Oh misread it, Model_type is read from model_config. Cool